xapian-glib/xapian-utils.cc

```diff
@@ -131,3 +131,33 @@ xapian_revision (void)
 {
   return Xapian::revision ();
 }
+
+/**
+ * xapian_sortable_serialize:
+ *
```

Comment:
Missing annotation for `value`; something like:

    * @value: the value to serialize